### PR TITLE
Using GPUS var to populate local CUDA_VISIBLE_DEVICES

### PR DESCRIPTION
**Pull request by hskarlupka**

### CODE REVIEW

File: pyglidein/startd_cron_scripts/clsim_gpu_test.py [hunk 1]
```diff
@@ -22,6 +21,15 @@ def main():
     (options, args) = parser.parse_args()
 
     try:
+        env = os.environ
+        gpus = env.get('GPUS', '0')
+        if gpus == '0':
+            print 'PYGLIDEIN_RESOURCE_GPU=False'
+            print '- update:true'
+            sys.exit(0)
```
Comment (dsschult) [inline pyglidein/startd_cron_scripts/clsim_gpu_test.py]: `sys.exit` is frowned upon.  would be better to restructure the code so this isn't necessary

File: pyglidein/startd_cron_scripts/clsim_gpu_test.py [hunk 2]
```diff
@@ -47,7 +55,7 @@ def main():
         print 'PYGLIDEIN_RESOURCE_GPU=True'
         print '- update:true'
     except SystemExit:
```
Comment (dsschult) [inline pyglidein/startd_cron_scripts/clsim_gpu_test.py]: How does `SystemExit` get generated?  This is an unusual exception to catch.

File: pyglidein/startd_cron_scripts/clsim_gpu_test.py (hunk 1 above)
Comment (hskarlupka) [reply pyglidein/startd_cron_scripts/clsim_gpu_test.py]: I forgot how much messing up I did to the script to add the crazy debugging logic.  I can clean this up a lot.

File: pyglidein/startd_cron_scripts/clsim_gpu_test.py (hunk 2 above)
Comment (hskarlupka) [reply pyglidein/startd_cron_scripts/clsim_gpu_test.py]: sys.exit() inside a try except statement will generate this.  This is going away so the exception handler will too.

**Pull request merged**